static/app/components/sidebar/index.tsx

```diff
@@ -634,9 +640,12 @@ const responsiveFlex = css`
   }
 `;
 
-export const SidebarWrapper = styled('nav')<{collapsed: boolean}>`
-  background: ${p => p.theme.sidebarGradient};
-  color: ${p => p.theme.sidebar.color};
+export const SidebarWrapper = styled('nav')<{collapsed: boolean; isSuperuser: boolean}>`
+  background: ${p =>
+    p.isSuperuser
+      ? 'linear-gradient(294.17deg,#8a2424 35.57%,#502636 92.42%,#502632 92.42%)'
```

Comment:
Eh it doesn't have to be this fancy. Just pick a red color that doesn't hurt the eye.